module/support/VirtualCamera/src/VirtualCamera.cpp

```diff
@@ -32,11 +32,9 @@ namespace support {
     using extension::Configuration;
 
     using message::input::Image;
-    using message::vision::LookUpTable;
 
     using utility::support::Expression;
-
-    using FOURCC = utility::vision::FOURCC;
+    using utility::vision::FOURCC;
```

Comment:
There is a new fourcc thing that came in with the new camera module that we should probably use instead of these constants, although maybe that is for another PR. (or maybe we should delete this whole module as i'm not sure it even works)

Reply:
If it doesn't work then it should be fixed. A fake camera module has its uses. Whatever we do, we can make that a later PR.